services/consumer/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"time"
+
+	"crypto/tls"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"k8s.io/klog"
+
+	pb "github.com/red-hat-storage/ocs-operator/services/provider/pb"
+)
+
+var (
+	storageProviderEndpoint = flag.String("endpoint", "localhost:50051", "The storageProviderEndpoint")
+	storageConsumerUID      = flag.String("storageConsumerUID", "UID", "The UID of storageConsumer CR")
```

Comment:
The default values here do not make sense. In addition all of this info can be read from within the Job which is something that we will need anyway when we start to report actual status

Reply:
Got it, updated in latest commit